src/util/tracker/autocompletes.ts

```diff
@@ -0,0 +1,182 @@
+import { AutocompleteInteraction } from "discord.js";
+import { activityTrackerBlacklistDb, activityTrackerLogDb } from "../../db";
+import { splitId } from "./help";
+
+export async function adminWhitelistgame(interaction: AutocompleteInteraction) {
+	if (!interaction.memberPermissions?.has("Administrator")) {
```

Comment:
Admin is quite a powerful perm. Something else might be a bit better, like manage server (which is the perm needed to add bots to the server) or maybe even moderate users, might work better.

Reply:
probably a good idea

Reply:
already made it customizable in [config](https://github.com/Cowoding-Jams/Jambo/blob/gameActivityTracker/src/config.ts#L60)